.gitignore

```diff
@@ -31,6 +31,7 @@ vendor
 
 # local resources, not to be committed
 local
+config/manager/kustomization.yaml
```

Comment:
is this intended?

Reply:
Yes, due to the `kustomization.yaml`  would be changed every time that we should deploy the new bundle.